action.yml

```diff
@@ -22,6 +22,9 @@ inputs:
     description: The domain of the Buf Schema Registry to login to.
     required: false
     default: 'buf.build'
+  go_version_file:
+    description: The go.mod file to read the buf version from.
```

Comment:
When would this ever be anything except go.mod? Why is this an option?

Reply:
Depending on the structure of the repo it may not be in the root.

It also supports just a raw version.